### DISCUSSION TITLE
Specify VPC instead of creating a new one

### DISCUSSION THREAD

**Discussion by corbinabeebe** [cdklabs/cdk-nextjs#38] [Q&A] [answered]
From what I can tell, it looks like I am limited in the ability to specify a vpc id to perform a look up vs. create a new vpc.  Can someone confirm this is the case, and if so, is there any plans to potentially add this as a prop?

Thanks~

**Comment by coltenkrauter** [comment, marked as answer]
Please check this example I made in an issue which shows how to use the override props to specify a custom VPC. https://github.com/cdklabs/cdk-nextjs/issues/16#issuecomment-2466572676

It also shows how to use [fck-nat](https://fck-nat.dev/), a much cheaper NAT solution.
